react/features/invite/components/add-people-dialog/web/CopyMeetingLinkSection.js

```diff
@@ -26,84 +26,14 @@ type Props = {
  * @returns {React$Element<any>}
  */
 function CopyMeetingLinkSection({ t, url }: Props) {
-    const [ isClicked, setIsClicked ] = useState(false);
-    const [ isHovered, setIsHovered ] = useState(false);
-
-    /**
-     * Click handler for the element.
-     *
-     * @returns {void}
-     */
-    function onClick() {
-        setIsHovered(false);
-        if (copyText(url)) {
-            setIsClicked(true);
-
-            setTimeout(() => {
-                setIsClicked(false);
-            }, 2500);
-        }
-    }
-
-    /**
-     * Hover handler for the element.
-     *
-     * @returns {void}
-     */
-    function onHoverIn() {
-        if (!isClicked) {
-            setIsHovered(true);
-        }
-    }
-
-    /**
-     * Hover handler for the element.
-     *
-     * @returns {void}
-     */
-    function onHoverOut() {
-        setIsHovered(false);
-    }
-
-    /**
-     * Renders the content of the link based on the state.
-     *
-     * @returns {React$Element<any>}
-     */
-    function renderLinkContent() {
-        if (isClicked) {
-            return (
-                <>
-                    <div className = 'invite-more-dialog copy-link-text selected'>
-                        {t('addPeople.linkCopied')}
-                    </div>
-                    <Icon src = { IconCheck } />
-                </>
-            );
-        }
-
-        const displayUrl = getDecodedURI(url);
-
-        return (
-            <>
-                <div className = 'invite-more-dialog invite-more-dialog-conference-url copy-link-text'>
```

Comment:
@quitrk The torture tests look for this selector: `invite-more-dialog-conference-url` that's why they are failing.